docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -420,10 +417,9 @@ CI Testing
         1. A dangerous module parameter has been deprecated or removed, and code is present to inform the user that they should not use this specific parameter anymore or that it stopped working intentionally.
         2. Module parameters are only used to pass in data from an accompanying action plugin.
 
-  * All entries in ignores.txt MUST have a justification in a comment in the ignore.txt file for each entry.  For example ``plugins/modules/docker_container.py use-argspec-type-path # uses colon-separated paths, can't use type=path``.
-  * Reviewers can block acceptance of a new collection if they don't agree with the ignores.txt entries.
+  * All entries in ignore-*.txt files MUST have a justification in a comment in the files for each entry.  For example ``plugins/modules/docker_container.py use-argspec-type-path # uses colon-separated paths, can't use type=path``.
```

Comment:
```suggestion
  * All entries in `ignore-*.txt` files MUST have a justification in a comment in the files for each entry.  For example `plugins/modules/docker_container.py use-argspec-type-path # uses colon-separated paths, can't use type=path`.
```

Reply:
I'm not an expert but i think two apostrophes would work better here